core/src/main/scala/monocle/Lens.scala

```diff
@@ -78,8 +78,8 @@ abstract class PLens[S, T, A, B] private[monocle]{ self =>
       def get(s: S): C =
         other.get(self.get(s))
 
-      def set(b: D): S => T =
-        self.modify(other.set(b))
+      def set(d: D): S => T =
+        self.modify(other.set(d))
```

Comment:
Thought of mentioning this when I prepared for my talk :)
The PR looks alright overall.